umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -99,6 +99,15 @@
     c
     (assoc c :RelatedUrls [u/not-provided-related-url])))
 
+(defn- migrate-spatialextent-down-from-1_18_0
+  "If SpatialExtent/SpatialCoverageType is EARTH/GLOBAL or LUNAR, remove the SpatialCoverageType."
```

Comment:
I don't remember, were we to remove the type or the entire spatial coverage?

Reply:
There is no such thing as entire spatial coverage.  SpatialExtent contains SpatialCoverageType and some other things.  SpatialCoverageType is the enum that we are adding some new values in 1.18.0 which doesn't exist in 1.17.3. So when migrating down,  the SpatialCoverageType needs to be removed if it contains those new values.